h2/src/main/org/h2/mvstore/tx/TransactionStore.java

```diff
@@ -233,7 +228,7 @@ public boolean hasMap(String name) {
      * @param logId the log id
      * @return the operation id
      */
-    static long getOperationId(int transactionId, long logId) {
+    private static long getOperationId(int transactionId, long logId) {
```

Comment:
This method should have default visibility as it has it before because it is accessed from an inner anonymous class.